src/service/API/Extensions/ServicesExtensions.cs

```diff
@@ -25,37 +29,49 @@ internal static class ServicesExtensions
         /// </summary>
         public static void AddAuthentication(this IServiceCollection services, IConfiguration configuration)
         {
-            services.AddAuthentication(JwtBearerDefaults.AuthenticationScheme)
-                   .AddJwtBearer(JwtBearerDefaults.AuthenticationScheme, options =>
-                   {
+            //services.AddAuthentication(JwtBearerDefaults.AuthenticationScheme)
```

Comment:
Remove these comments if it is not required.

Reply:
Removed the comments.